### PR TITLE
Add comment to review

### PR DESCRIPTION
**Pull request by hieuhtk58**

### CODE REVIEW

File: app/views/comments/_comment_form.html.erb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<%= form_for [@book, @review, @comment] do |f| %>
+
+<%= f.text_field :content %>
```
Comment (ngotungson) [inline app/views/comments/_comment_form.html.erb]: ident

File: app/views/reviews/_review.html.erb [hunk 2]
```diff
@@ -1,6 +1,11 @@
 <div><%= review.rating %></div>
 <div><%= review.content %></div>
 <div>
+  <%= link_to t("views.book.show.comment"),
+    new_book_review_comment_path(review.book, review) %>
+</div>
+<%= render review.comments %>
+<div>
```
Comment (ngotungson) [inline app/views/reviews/_review.html.erb]: thiếu đóng div

**Comment by ngotungson** [comment]
ok

**Comment by phanbt58** [comment]
OK

**Pull request merged**